src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,134 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.exceptions.DukeException;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+
+public class Storage {
+    private final File f;
```

Comment:
This might be minor, but do you think naming the variable as "file" instead of "f" would improve readability?

Reply:
noted, thanks edeline!!